From a076b09326c84a23c0174fb94b7e82bbb1f24a44 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Thu, 22 Aug 2019 16:53:19 -0700
Subject: [PATCH 300/830] ANDROID: Revert "um: irq: don't set the chip for all
 irqs"
X-NVConfidentiality: public

This reverts commit 1987b1b8f9f17a06255877e7917d0bb5b5377774.

Reason: Broke UML used by kernel_tests

Bug: 139897923
Change-Id: If3541721fdca7cf6d77410309ae5b503b5a848d0
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/um/kernel/irq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/um/kernel/irq.c b/arch/um/kernel/irq.c
index 3577118bb4a5..9410424af710 100644
--- a/arch/um/kernel/irq.c
+++ b/arch/um/kernel/irq.c
@@ -480,7 +480,7 @@ void __init init_IRQ(void)
 	irq_set_chip_and_handler(TIMER_IRQ, &SIGVTALRM_irq_type, handle_edge_irq);
 
 
-	for (i = 1; i <= LAST_IRQ; i++)
+	for (i = 1; i < NR_IRQS; i++)
 		irq_set_chip_and_handler(i, &normal_irq_type, handle_edge_irq);
 	/* Initialize EPOLL Loop */
 	os_setup_epoll();
-- 
2.17.1

